# The goal message
float64 x
float64 y

---
# The result message
bool success
float64 elapsed_time

---
# The feedback message
float64 distance_to_goal
float64 linear_velocity
float64 angular_velocity